### PR TITLE
Edited deploy-ydb-federated-query.md

### PR DESCRIPTION
**Pull request by anton-bobkov**
### Changelog category

* Documentation (changelog entry is not required)

### Additional information

Edited the existing topic. Nothing new was added.

### CODE REVIEW

File: ydb/docs/en/core/deploy/manual/deploy-ydb-federated-query.md [hunk 1]
```diff
@@ -56,4 +62,4 @@ Currently, we do not support deploying the connector in {{ k8s }}, but we plan t
         enable_external_data_sources: true
         enable_script_execution_operations: true
     ```
-5. Continue deploying the dynamic node {{ ydb-short-name }} according to the [instructions](./deploy-ydb-on-premises.md).
\ No newline at end of file
+5. Continue deploying the dynamic node {{ ydb-short-name }}. See the [instructions](./deploy-ydb-on-premises.md).
```
[the remark this answers is not in the document]
Comment (blinkov) [reply ydb/docs/en/core/deploy/manual/deploy-ydb-federated-query.md:65]: If I had to choose, I'd prefer the previous "according to" version, but both are fine.

File: ydb/docs/en/core/deploy/manual/deploy-ydb-federated-query.md [hunk 2]
```diff
@@ -19,8 +19,14 @@ A special microservice called [connector](../../concepts/federated_query/archite
...
-* The external data source must be accessible over the network for queries from {{ ydb-short-name }} or from the connector (if present);
-* The connector must be accessible over the network for queries from {{ ydb-short-name }} (which is trivially achieved by running these processes on the same host).
+* The external data source must be accessible over the network to queries from {{ ydb-short-name }} or from the connector, if present.
+* The connector must be accessible over the network to queries from {{ ydb-short-name }}.
+  
+  {% note tip %}
+  
+  To make the connector accessible to the queries, run these processes on the same host.
+  
+  {% endnote %}
```
Comment (blinkov) [inline ydb/docs/en/core/deploy/manual/deploy-ydb-federated-query.md:29]:
```suggestion
* The external data source must be accessible over the network to queries from {{ ydb-short-name }} database nodes or from the connector, if present.
* The connector must be accessible over the network from {{ ydb-short-name }} database nodes.
  
 {% note tip %}
  
 The easiest way to make the connector accessible from {{ ydb-short-name }} nodes is to run them on the same set of hosts.
  
 {% endnote %}
```

**Review by blinkov — DISMISSED** [review]

File: ydb/docs/en/core/deploy/manual/deploy-ydb-federated-query.md [hunk 3]
```diff
@@ -56,4 +62,4 @@ Currently, we do not support deploying the connector in {{ k8s }}, but we plan t
         enable_external_data_sources: true
         enable_script_execution_operations: true
     ```
-5. Continue deploying the dynamic node {{ ydb-short-name }} according to the [instructions](./deploy-ydb-on-premises.md).
\ No newline at end of file
+5. Continue deploying {{ ydb-short-name }} database nodes. See the [instructions](./deploy-ydb-on-premises.md).
```
Comment (fomichev3000) [inline ydb/docs/en/core/deploy/manual/deploy-ydb-federated-query.md:65]:
```suggestion
5. Continue deploying {{ ydb-short-name }} dynamic nodes. See the [instructions](./deploy-ydb-on-premises.md).
```

**Review by fomichev3000 — APPROVED** [review]
Approve, but consider my comment. 'Dynamic node' is not a synonym for a 'database node'.

**Pull request merged**